audb/core/publish.py

```diff
@@ -67,21 +104,26 @@ def _find_tables(
 def _find_media(
         db: audformat.Database,
         db_root: str,
+        db_root_files: typing.Set[str],
         version: str,
         deps: Dependencies,
         archives: typing.Mapping[str, str],
         num_workers: int,
         verbose: bool,
 ) -> typing.Set[str]:
+    r"""Find archives with new, altered or removed media files."""
 
     # release dependencies to removed media
     # and select according archives for upload
-    media = set()
-    db_media = db.files
-    for file in set(deps.media) - set(db_media):
-        media.add(deps.archive(file))
+    media_archives = set()
+    db_media = set(db.files)
+    for file in set(deps.media) - db_media:
+        media_archives.add(deps.archive(file))
         deps._drop(file)
```

Comment:
To increase readability I would change to

```python
    media_archives = set()
    db_media = set(db.files)

    # release dependencies to removed media
    # and select according archives for upload
    removed_media = set(deps.media) - db_media
    for file in removed_media:
        media_archives.add(deps.archive(file))
        deps._drop(file)
```

Reply:
done